src/renderers/WebGLRenderTargetCube.js

```diff
@@ -15,5 +16,97 @@ WebGLRenderTargetCube.prototype.constructor = WebGLRenderTargetCube;
 
 WebGLRenderTargetCube.prototype.isWebGLRenderTargetCube = true;
 
+WebGLRenderTargetCube.prototype.fromEquirectangularTexture = function ( renderer, texture ) {
+
+	this.texture.type = texture.type;
+	this.texture.format = texture.format;
+	this.texture.encoding = texture.encoding;
+
+	var scene = new THREE.Scene();
```

Comment:
I've noticed today that `fromEquirectangularTexture()` uses the `THREE` namespace at several places. Ideally classes like `Scene`, `ShaderMaterial` etc. are imported into the module.

Reply:
Rats! Sorry,  I'll fix.

Reply:
#16819